src/etc/inc/services.inc

```diff
@@ -174,8 +174,10 @@ function services_radvd_configure($blacklist = array())
 
         $stanzas = array();
         $ifcfgsnv6 = get_interface_subnetv6($dhcpv6if);
-        $subnetv6 = gen_subnetv6($ifcfgipv6, $ifcfgsnv6);
-        $stanzas[] = "{$subnetv6}/{$ifcfgsnv6}";
+        if ($ifcfgsnv6) {
```

Comment:
can we make this !empty() ... should be the same?

Reply:
Whatever way floats your boat... :)